src/main.rs

```diff
@@ -94,6 +95,9 @@ fn main() {
         .arg(
             flag_arg("prioritize tasks", 'z', "prioritize", "Assign priorities to tasks. Can specify project option, defaults to inbox.")
         )
+        .arg(
+            flag_arg("edit tasks", 'd', "edit", "Edit tasks. Can specify project option, defaults to inbox.")
```

Comment:
So far I haven't found a good way to test this app from the outside, but want to improve that in the future.

For now I also have `manual_test.sh` which makes sure that the CLI commands are still intact and working as intended :+1: